sdk/appcenter-distribute/src/main/java/com/microsoft/appcenter/distribute/Distribute.java

```diff
@@ -1768,4 +1801,31 @@ synchronized void setInstalling(ReleaseDetails releaseDetails) {
             PreferencesStorage.putInt(PREFERENCE_KEY_DOWNLOAD_STATE, DOWNLOAD_STATE_INSTALLING);
         }
     }
+
+    /**
+     * Send distribution start session log after enabling in-app updates (first app launch after installation).
+     */
+    private synchronized void enqueueDistributionStartSessionLog() {
+
+        /*
+         * Session starts before in-app updates setup (using browser) so the first start session log
+         * is sent without distributionGroupId value.
+         *
+         * Send the distribution start session log if start session log without distributionGroupId
+         * value was sent before
+         */
+        SessionContext.SessionInfo lastSession = SessionContext.getInstance().getSessionAt(System.currentTimeMillis());
+        if (lastSession == null) {
```

Comment:
lastSession can have null sessionid if analytics disabled, you need to test that also (a launch session is created by the core before analytics adds one)

Reply:
Fixed. Thank you.